RationaleMCP/0031/grammar.md

```diff
@@ -173,6 +174,24 @@ end _F;
 > ~~_import-list_ → _IDENT_ ( **,** _IDENT_ )*~~
 
 
+## Clock partitions
+
+> _base-partition_ →\
+> &emsp; **partition** _comment_\
```

Comment:
Note the presence of _comment_ here.  It opens up for giving a description string and an annotation for the base-partition.  Note that placing an annotation at the end of the base-partition would be problematic due to risk of mixing it up with annotations belonging to the model or the last sub-partition inside the base-partition.

Reply:
This was brought to attention in today's meeting.  Resolving.